weblate/api/views.py

```diff
@@ -934,16 +934,34 @@ def destroy(self, request, *args, **kwargs):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
-class LanguageViewSet(viewsets.ReadOnlyModelViewSet):
+class LanguageViewSet(viewsets.ModelViewSet):
     """Languages API."""
 
     queryset = Language.objects.none()
     serializer_class = LanguageSerializer
     lookup_field = "code"
 
     def get_queryset(self):
+        if self.request.user.is_superuser:
```

Comment:
I think it makes more sense to check the actual permission:

```suggestion
        if request.user.has_perm("language.edit"):
```